src/ansys/aedt/core/workflows/circuit/toolkits_catalog.toml

```diff
@@ -2,5 +2,5 @@
 name = "Schematic Importer"
 script = "import_schematic.py"
 icon = "images/large/schematic.png"
-template = "run_pyedb_toolkit_script"
```

Comment:
What is the reason to remove `run_pyedb_toolkit_script` file ? Was it too much to maintain ? Also, if there was a value to have both file, do we still have the same "features" as when the files were splitted ?

Reply:
After some tests, we realized RunProgram (which was used in run_pyaedt) is not needed, and we can use directly subprocess for both Windows and Linux. The only difference between pyedb and pyaedt script templates was this, so now we just need one template.